less/classbxr.less

```diff
@@ -12,9 +12,9 @@
     background-color: white;
     color: cornflowerblue;
     position: absolute;
-    top: -.5em;
+    top: -1.5ex;
     white-space: nowrap;
-    font-family: sans-serif;
+    font-family: monospace;
```

Comment:
The thing about mono is that it's wider - I'd like to keep this as tight as possible so it minimizes the disruption to the layout.